x/auth/client/cli/cli_test.go

```diff
@@ -985,6 +986,50 @@ func (s *IntegrationTestSuite) TestTxWithoutPublicKey() {
 	s.Require().NoError(err)
 }
 
+func (s *IntegrationTestSuite) TestFlagOutputDocument() {
+	val0 := s.network.Validators[0]
+	res, err := bankcli.MsgSendExec(
+		val0.ClientCtx,
+		val0.Address,
+		val0.Address,
+		sdk.NewCoins(
+			sdk.NewCoin(fmt.Sprintf("%stoken", val0.Moniker), sdk.NewInt(10)),
+			sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10)),
+		),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+		fmt.Sprintf("--%s=true", flags.FlagGenerateOnly),
+	)
+	s.Require().NoError(err)
+
+	// Write  unsigned tx to file.
+	unsignedTx, cleanup1 := testutil.WriteToNewTempFile(s.T(), res.String())
+	defer cleanup1()
+
+	// Create a temp file to hold the signed tx.
+	tempfile, err := ioutil.TempFile("", "testtx")
```

Comment:
Alternatively, you could do this:

```suggestion
	tempfile, err := ioutil.TempFile(s.T().TempDir(), "testtx")
```

The file would be automatically purged along with the temporary directory at the end of the test case (thus no need to call `os.Remove()`) :+1:

Reply:
Nice! I did this in 04c41fbffc438098307045f6e7004c1a55ddd5b6